.github/workflows/pull-request.yml

```diff
@@ -377,25 +376,40 @@ jobs:
       #     path: "./coverage/vitest/lcov.info"
       #     min_coverage: 86.6
 
-  Test-Docusaurus-Deployment:
-    name: Test Deployment to https://docs-plugin.talawa.io
+  Docs-Checks-and-Build:
```

Comment:
1. No changes must be made to the Docusaurus test deployment section of the workflow. You have changed its purpose which is not in scope.
2. All changes must be made to the existing code quality workflow.
3. You can run linting and type checking specifying the directory on the command line. There is no need to duplicate this work. The task is very simple.